src/Microsoft.ApplicationInsights/Channel/IAiSerializableTelemetry.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Microsoft.ApplicationInsights.Channel
+{
+    using System.Collections.Generic;
+
+    public interface IAiSerializableTelemetry
```

Comment:
Shouldn't this be `internal`?

Reply:
Can we do that?
I thought it would be discouraged to have internal interfaces on public classes.

If we can do that, we absolutely should. :)